person.rb

```diff
@@ -1,8 +1,11 @@
-class Person
+require 'nameablee'
+
+class Person < Nameable
   attr_reader :id
   attr_accessor :name, :age
 
   def initialize(name, age)
+    super()
     @id = Time.now.to_f
     @name = name
     @age = age
```

Comment:
- [ ] According to the project requirements, The `Person` class inherits from the `Nameable` class. should also have a method `correct_name`. Kindly add this 👍🏻 

```ruby
class Person < Nameable
  .
  .
    def correct_name
       name
    end
  .
  .
end
```
```